browser/html/cool-help.html

```diff
@@ -6,573 +6,768 @@
         <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1363).scrollIntoView();'>Formulas</button></p>
-        <p class="toc-h3 m-v-0"><button class="border-0" honClick='document.getElementById(1367).scrollIntoView();'>Formatting spreadsheets</button></p>
+        <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1367).scrollIntoView();'>Formatting spreadsheets</button></p>
         <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1371).scrollIntoView();'>Advanced features</button></p>
+        <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1409).scrollIntoView();'>Spreadsheets</button></p>
     </div>
-    <div class="text" style="display: none;">
+    <div class="text link-section" style="display: none;">
         <p class="toc-h2 m-v-0"><button class="border-0" onClick='document.getElementById(1373).scrollIntoView();'><span class="productname">%productName</span> text documents</button></p>
         <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1375).scrollIntoView();'>Editing text documents</button></p>
         <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1377).scrollIntoView();'>Context menus</button></p>
         <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1383).scrollIntoView();'>Advanced text document editor features</button></p>
+        <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1407).scrollIntoView();'>Text documents</button></p>
     </div>
-    <div class="presentation" style="display: none;">
+    <div class="presentation link-section" style="display: none;">
         <p class="toc-h2 m-v-0"><button class="border-0" onClick='document.getElementById(1391).scrollIntoView();'><span class="productname">%productName</span> presentations</button></p>
         <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1393).scrollIntoView();'>Editing presentations</button></p>
         <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1395).scrollIntoView();'>Slide show</button></p>
         <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1397).scrollIntoView();'>Slide pane</button></p>
         <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1401).scrollIntoView();'>Advanced features</button></p>
+        <p class="toc-h3 m-v-0"><button class="border-0" onClick='document.getElementById(1411).scrollIntoView();'>Presentations</button></p>
     </div>
     <p class="toc-h2 m-v-0"><button class="border-0" onClick='document.getElementById(1403).scrollIntoView();'>Frequently Asked Questions</button></p>
```

Comment:
no section for FAQ?

Reply:
No need for separate section it is with the common links as you can see at start of that div there are some common links which does not need section. We will anyway hide that links when we start filter content.